bin/describe.ml

```diff
@@ -170,23 +170,47 @@ module Descr = struct
         ]
   end
 
+  module Root = struct
+    type t = Path.t
+
+    let[@warning "-27"] map_path (_ : t) ~f =
+      Path.external_ (Path.External.of_string "/")
+
+    let to_dyn t = Dyn.string (Path.to_absolute_filename t)
+  end
+
+  module BuildCtxt = struct
```

Comment:
Please add a comment at the beginning of this module (similar to the comments for its brother modules).